docker-compose.yaml

```diff
@@ -1,8 +1,18 @@
 version: "3"
 services:
   aspnet:
-    build:
-      context: .
+    image: wayhowen/devoops:latest
```

Comment:
technically we should use a version and not the latest, since with latest we can introduce some unwanted changes that will always get pulled, and with a set version we can test it locally before updating

Reply:
If it hasn't been tested/isn't stable it probably shouldn't be added to the hub